docs/content/seafile.md

```diff
@@ -0,0 +1,242 @@
+s) Set configuration password
+q) Quit config
+n/s/q> n
+name> seafile
+Type of storage to configure.
+Enter a string value. Press Enter for the default ("").
+Choose a number from below, or type in your own value
+[snip]
+XX / Seafile
+   \ "seafile"
+[snip]
+Storage> seafile
+** See help for seafile backend at: https://rclone.org/seafile/ **
+
+URL of seafile host to connect to
+Enter a string value. Press Enter for the default ("").
+Choose a number from below, or type in your own value
+ 1 / Connect to cloud.seafile.com
+   \ "https://cloud.seafile.com/"
+url> http://my.seafile.server/
+User name
+Enter a string value. Press Enter for the default ("").
+user> me@example.com
+Password
+y) Yes type in my own password
+g) Generate random password
+y/g> y
+Enter the password:
+password:
+Confirm the password:
+password:
+Name of the library. Leave blank to access all non-encrypted libraries.
+Enter a string value. Press Enter for the default ("").
+library> My Library
+Library password (for encrypted libraries only). Leave blank if you pass it through the command line.
+y) Yes type in my own password
+g) Generate random password
+n) No leave this optional password blank (default)
+y/g/n> n
+Edit advanced config? (y/n)
+y) Yes
+n) No (default)
+y/n> n
+Remote config
+--------------------
+[seafile]
+type = seafile
+url = http://my.seafile.server/
+user = me@example.com
+password = *** ENCRYPTED ***
+library = My Library
+--------------------
+y) Yes this is OK (default)
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+You specified `My Library` during the configuration. The root of the remote is pointing at the
+root of the library `My Library`:
+
+See all files in the library:
+
+    rclone lsd seafile:
+
+Create a new directory inside the library
+
+    rclone mkdir seafile:directory
+
+List the contents of a directory
+
+    rclone ls seafile:directory
+
+Sync `/home/local/directory` to the remote library, deleting any
+excess files in the library.
+
+    rclone sync /home/local/directory seafile:
+
+
+### --fast-list ###
+
+Seafile version 7+ supports `--fast-list` which allows you to use fewer
+transactions in exchange for more memory. See the [rclone
+docs](/docs/#fast-list) for more details.
+Please note this is not supported on seafile server version 6.x
+
+
+#### Restricted filename characters
+
+In addition to the [default restricted characters set](/overview/#restricted-characters)
+the following characters are also replaced:
+
+| Character | Value | Replacement |
+| --------- |:-----:|:-----------:|
+| /         | 0x2F  | ／          |
+| "         | 0x22  | ＂          |
+| \         | 0x5C  | ＼           |
+
+Invalid UTF-8 bytes will also be [replaced](/overview/#invalid-utf8),
+as they can't be used in JSON strings.
+
+### Seafile and rclone link ###
+
+Rclone supports generating share links for non-encrypted libraries only.
+They can either be for a file or a directory:
+
+```
+./rclone link seafile:seafile-tutorial.doc
```

Comment:
Can we lose the `./` at the start of the example as the rest of the docs assume the user has rclone on the PATH?